src/thumbs.jsx

```diff
@@ -0,0 +1,35 @@
+/*
+thumbs are 56x56px w/ border radius ~5px
+hovering over thumb puts a 2px black border and changes the image to the active in carosel
+unselected thumbs have a white opaque overlay to fade it slightly
+thumbs are aligned vertically along left side of screen, non-responsive
+*/
+
+import React from 'react';
+import './styles/styles.css';
+
+class Thumbs extends React.Component {
+  constructor(props) {
+    super(props);
+  }
```

Comment:
Could probably switch this to being a functional component since there's no state here, unless you had future plans otherwise.